### ISSUE TITLE
failed to start ddev-router due to `apk` failing building Traefik router (VPN)

### ISSUE THREAD

**Issue by keithdoyle9** [ddev/ddev#5209]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Output of `ddev debug test`

Upgraded to v1.22.0 today and am getting the following error:
```
RUN apk add bash curl htop vim:
0.260 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
0.260 ERROR: unable to select packages:
0.260   bash (no such package):
0.260     required by: world[bash]
0.260   curl (no such package):
0.260     required by: world[curl]
0.260   htop (no such package):
0.260     required by: world[htop]
0.260   vim (no such package):
0.260     required by: world[vim]
```

config.yaml
```
name: seic-enterprise
type: drupal9
docroot: docroot
php_version: '8.1'
webserver_type: apache-fpm
router_http_port: '80'
router_https_port: '443'
xdebug_enabled: false
additional_hostnames: []
additional_fqdns: []
database:
    type: mysql
    version: '5.7'
nfs_mount_enabled: false
hooks:
    post-import-db:
        - exec: drush deploy
    post-start:
        - exec: acli auth:login --key $ACQUIA_API_KEY --secret $ACQUIA_API_SECRET -n
        - composer: install
        - exec-host: ddev auth ssh
use_dns_when_possible: true
composer_version: '2'
webimage_extra_packages: [make, g++]
web_environment:
    - NODE_EXTRA_CA_CERTS=/var/www/html/sei-combined.crt
    - AH_SITE_ENVIRONMENT=DDEV
web_extra_exposed_ports:
    - name: storybook
      container_port: 6006
      http_port: 4000
      https_port: 3999
```
Contents of docker file:
```
ARG BASE_IMAGE
FROM $BASE_IMAGE
ADD *.crt /usr/local/share/ca-certificates/
RUN update-ca-certificates --fresh
RUN curl -OL https://github.com/acquia/cli/releases/latest/download/acli.phar && \
    chmod +x acli.phar && \
    mv acli.phar /usr/local/bin/acli
```
<details><summary>Expand `ddev debug test` diagnostic information</summary>

```
Running bash [-c /var/folders/pt/wgs8tvgd7m72dd1935hsy2b80000gn/T/test_ddev.sh] 
======= Existing project config =========
These config files were loaded for project seic-enterprise: [/Users/kdoyle/Code/seic-enterprise/.ddev/config.yaml]
name: seic-enterprise
type: drupal9
docroot: docroot
php_version: 8.1
webserver_type: apache-fpm
webimage: ddev/ddev-webserver:v1.22.0
router_http_port: 80
router_https_port: 443
additional_hostnames: []
additional_fqdns: []
database: {mysql 5.7}
hooks: map[post-import-db:[map[exec:drush deploy]] post-start:[map[exec:acli auth:login --key $ACQUIA_API_KEY --secret $ACQUIA_API_SECRET -n] map[composer:install] map[exec-host:ddev auth ssh]]]
mailhog_port: 8025
mailhog_https_port: 8026
webimage_extra_packages: [make g++]
project_tld: ddev.site
use_dns_when_possible: true
composer_version: 2
nodejs_version: 18
default_container_timeout: 120
web_extra_exposed_ports: [{storybook 6006 4000 3999}]
======= Creating dummy project named  tryddevproject-21037 in ../tryddevproject-21037 =========
OS Information: Darwin MAC-F9T276DYX3 22.4.0 Darwin Kernel Version 22.4.0: Mon Mar  6 20:59:28 PST 2023; root:xnu-8796.101.5~3/RELEASE_ARM64_T6000 arm64
ProductName:            macOS
ProductVersion:         13.3.1
ProductVersionExtra:    (a)
BuildVersion:           22E772610a
User information: uid=501(kdoyle) gid=20(staff) groups=20(staff),12(everyone),61(localaccounts),79(_appserverusr),80(admin),81(_appserveradm),98(_lpadmin),33(_appstore),100(_lpoperator),204(_developer),250(_analyticsusers),395(com.apple.access_ftp),398(com.apple.access_screensharing),399(com.apple.access_ssh),400(com.apple.access_remote_ae),701(com.apple.sharepoint.group.1)
DDEV version:  ITEM             VALUE                                   
 DDEV version     v1.22.0                                 
 architecture     arm64                                   
 db               ddev/ddev-dbserver-mariadb-10.4:v1.22.0 
 ddev-ssh-agent   ddev/ddev-ssh-agent:v1.22.0             
 docker           24.0.2                                  
 docker-compose   v2.20.0                                 
 docker-platform  docker-desktop                          
 mutagen          0.17.2                                  
 os               darwin                                  
 router           traefik:v2.10                           
 web              ddev/ddev-webserver:v1.22.0             
PROXY settings: HTTP_PROXY='' HTTPS_PROXY='' http_proxy='' NO_PROXY=''
======= DDEV global info =========
Global configuration:
instrumentation-opt-in=true
omit-containers=[]
performance-mode=mutagen
router-bind-all-interfaces=false
internet-detection-timeout-ms=3000
disable-http2=false
use-letsencrypt=false
letsencrypt-email=
table-style=default
simple-formatting=false
auto-restart-containers=false
use-hardened-images=false
fail-on-hook-fail=false
required-docker-compose-version=v2.20.0
use-docker-compose-from-path=false
project-tld=
xdebug-ide-location=
no-bind-mounts=false
router=traefik
wsl2-no-windows-hosts-mgt=false
router-http-port=80
router-https-port=443

======= DOCKER info =========
docker location: lrwxr-xr-x  1 root  wheel  54 Jan 12  2023 /usr/local/bin/docker -> /Applications/Docker.app/Contents/Resources/bin/docker
Docker Desktop Version: Docker Desktop for Mac 4.21.1 build 114176
docker version: 
Client:
 Cloud integration: v1.0.35
 Version:           24.0.2
 API version:       1.43
 Go version:        go1.20.4
 Git commit:        cb74dfc
 Built:             Thu May 25 21:51:16 2023
 OS/Arch:           darwin/arm64
 Context:           default

Server: Docker Desktop 4.21.1 (114176)
 Engine:
  Version:          24.0.2
  API version:      1.43 (minimum version 1.12)
  Go version:       go1.20.4
  Git commit:       659604f
  Built:            Thu May 25 21:50:59 2023
  OS/Arch:          linux/arm64
  Experimental:     false
 containerd:
  Version:          1.6.21
  GitCommit:        3dce8eb055cbb6872793272b4f20ed16117344f8
 runc:
  Version:          1.1.7
  GitCommit:        v1.1.7-0-g860f061
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
DOCKER_DEFAULT_PLATFORM=notset
======= Mutagen Info =========
Mutagen is installed in ddev, version=0.17.2
--------------------------------------------------------------------------------
Name: seic-enterprise
Identifier: sync_lmAR7drqFXl3v94GxzsJpCNxJTUHuEpd2th5qe3ypxL
Labels:
        com.ddev.config-hash: 654a7799e053320033f4baa45c0f9c345f2546e4
        com.ddev.volume-signature: Users-kdoyle-docker-run-1690551112
Configuration:
        Synchronization mode: Two Way Resolved
        Hashing algorithm: Default (SHA-1)
        Maximum allowed entry count: Default (2⁶⁴−1)
        Maximum staging file size: Default (18 EB)
        Symbolic link mode: POSIX Raw
        Ignore VCS mode: Default (Propagate)
        Ignores:
                /.git
                /.tarballs
                /.ddev/db_snapshots
                /.ddev/.importdb*
                .DS_Store
                .idea
                /docroot/sites/default/files
        Permissions mode: Default (Portable)
Alpha:
        URL: /Users/kdoyle/Code/seic-enterprise
        Configuration:
                Watch mode: Default (Portable)
                Watch polling interval: Default (10 seconds)
                Probe mode: Default (Probe)
                Scan mode: Default (Accelerated)
                Stage mode: Neighboring
                File mode: Default (0600)
                Directory mode: Default (0700)
                Default file/directory owner: Default
                Default file/directory group: Default
        Connected: Yes
        Synchronizable contents:
                32716 directories
                273929 files (1.1 GB)
                693 symbolic links
Beta:
        URL: docker://ddev-seic-enterprise-web/var/www/html
                DOCKER_HOST=unix:///Users/kdoyle/.docker/run/docker.sock
        Configuration:
                Watch mode: Default (Portable)
                Watch polling interval: Default (10 seconds)
                Probe mode: Default (Probe)
                Scan mode: Default (Accelerated)
                Stage mode: Neighboring
                File mode: Default (0600)
                Directory mode: Default (0700)
                Default file/directory owner: Default
                Default file/directory group: Default
                Compression: Default (DEFLATE)
        Connected: Yes
        Synchronizable contents:
                32716 directories
                273929 files (1.1 GB)
                693 symbolic links
Status: Watching for changes
--------------------------------------------------------------------------------
======= Docker Info =========
Docker platform: docker-desktop 
Using docker context: default (unix:///Users/kdoyle/.docker/run/docker.sock) 
docker-compose: v2.20.0 
Using DOCKER_HOST=unix:///Users/kdoyle/.docker/run/docker.sock 
Docker version: 24.0.2 
Able to run simple container that mounts a volume. 
Docker disk space:
Filesystem                Size      Used Available Use% Mounted on
overlay                  95.0G     10.0G     80.2G  11% /

unable to create project at project root '/Users/kdoyle/Code/seic-code-studio': project root /Users/kdoyle/Code/seic-code-studio does not exist
 Container ddev-seic-enterprise-db  Stopped
 Container ddev-seic-enterprise-web  Stopped
 Container ddev-seic-enterprise-db  Stopped
 Container ddev-seic-enterprise-web  Stopped
 Container ddev-seic-enterprise-db  Removed
 Container ddev-seic-enterprise-web  Removed
 Network ddev-seic-enterprise_default  Removed
Project seic-enterprise has been stopped.
Stopped mutagen daemon
The ddev-ssh-agent container has been removed. When you start it again you will have to use 'ddev auth ssh' to provide key authentication again.
Existing docker containers: 
CONTAINER ID   IMAGE     COMMAND   CREATED   STATUS    PORTS     NAMES
Network ddev_default created 
Starting tryddevproject-21037... 
 Container ddev-ssh-agent  Created 
 Container ddev-ssh-agent  Started 
ssh-agent container is running: If you want to add authentication to the ssh-agent container, run 'ddev auth ssh' to enable your keys. 
v1.22.0: Pulling from ddev/ddev-dbserver-mariadb-10.4
1e77eb131c5c: Pulling fs layer 
5b639cf53b90: Pulling fs layer 
16e3566491c6: Pulling fs layer 
4b2bae540b3f: Waiting 
687ebf328168: Waiting 
d25233c95c0a: Pulling fs layer 
273b09ea8e09: Pulling fs layer 
acf6c1b3fad9: Pulling fs layer 
d25233c95c0a: Waiting 
4f4fb700ef54: Pull complete 
334f7933b4a6: Pull complete 
1f6409eead7b: Pull complete 
2ff9ee533a67: Pull complete 
e756920b1b90: Pull complete 
3d507dd19929: Pull complete 
6a51dc6645b3: Pull complete 
9d0a0d58a087: Pull complete 
02f691e9286c: Pull complete 
e0363f9c301e: Pull complete 
8cbc80cf5da6: Pull complete 
f85c924b561d: Pull complete 
dd5c4d757aac: Pull complete 
81dd35a721df: Pull complete 
5eb58dc028d6: Pull complete 
Digest: sha256:aecfb2180c10c1af0f4503904b44839cd38c8183b69d9428863ddafa14f8b2c1
Status: Downloaded newer image for ddev/ddev-dbserver-mariadb-10.4:v1.22.0
docker.io/ddev/ddev-dbserver-mariadb-10.4:v1.22.0
 Network ddev-tryddevproject-21037_default  Created 
 Container ddev-tryddevproject-21037-db  Created 
 Container ddev-tryddevproject-21037-web  Created 
 Container ddev-tryddevproject-21037-web  Started 
 Container ddev-tryddevproject-21037-db  Started 
You have Mutagen enabled and your 'php' project type doesn't have `upload_dirs` set. 
For faster startup and less disk usage, set upload_dirs to where your user-generated files are stored. 
If this is intended you can disable this warning with `ddev config --disable-upload-dirs-warning`. 
Starting mutagen sync process... This can take some time. 
Mutagen sync flush completed in 2s.
For details on sync status 'ddev mutagen st tryddevproject-21037 -l' 
failed to solve: process "/bin/sh -c apk add bash curl htop vim" did not complete successfully: exit code: 4 
Failed to start tryddevproject-21037: failed to start ddev-router: ComposeCmd failed to run 'COMPOSE_PROJECT_NAME=ddev-tryddevproject-21037 docker-compose -f /Users/kdoyle/.ddev/.router-compose-full.yaml -p ddev-router up --build -d', action='[-p ddev-router up --build -d]', err='exit status 17', stdout='#1 [ddev-router internal] load .dockerignore
#1 transferring context: 2B done
#1 DONE 0.0s

#2 [ddev-router internal] load build definition from Dockerfile
#2 transferring dockerfile: 206B done
#2 DONE 0.0s

#3 [ddev-router internal] load metadata for docker.io/library/traefik:v2.10
#3 DONE 0.0s

#4 [ddev-router 1/4] FROM docker.io/library/traefik:v2.10
#4 CACHED

#5 [ddev-router internal] load build context
#5 transferring context: 1.07kB done
#5 DONE 0.0s

#6 [ddev-router 2/4] RUN apk add bash curl htop vim
#6 0.296 fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/main/aarch64/APKINDEX.tar.gz
#6 0.733 fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/community/aarch64/APKINDEX.tar.gz
#6 0.735 60900EBEFFFF0000:error:0A000086:SSL routines:tls_post_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1889:
#6 0.735 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/main: Permission denied
#6 1.100 60900EBEFFFF0000:error:0A000086:SSL routines:tls_post_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1889:
#6 1.100 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
#6 1.100 ERROR: unable to select packages:
#6 1.100   bash (no such package):
#6 1.100     required by: world[bash]
#6 1.100   curl (no such package):
#6 1.100     required by: world[curl]
#6 1.100   htop (no such package):
#6 1.100     required by: world[htop]
#6 1.100   vim (no such package):
#6 1.100     required by: world[vim]
#6 ERROR: process "/bin/sh -c apk add bash curl htop vim" did not complete successfully: exit code: 4
------
 > [ddev-router 2/4] RUN apk add bash curl htop vim:
1.100 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
1.100 ERROR: unable to select packages:
1.100   bash (no such package):
1.100     required by: world[bash]
1.100   curl (no such package):
1.100     required by: world[curl]
1.100   htop (no such package):
1.100     required by: world[htop]
1.100   vim (no such package):
1.100     required by: world[vim]
------
', stderr='failed to solve: process "/bin/sh -c apk add bash curl htop vim" did not complete successfully: exit code: 4' 
unable to create project at project root '/Users/kdoyle/Code/seic-code-studio': project root /Users/kdoyle/Code/seic-code-studio does not exist
┌──────────────────────┬──────────────┬─────────────────────────────┬─────────────────────────── ≈
│ NAME                 │ STATUS       │ LOCATION                    │ URL                        ≈
├──────────────────────┼──────────────┼─────────────────────────────┼─────────────────────────── ≈
│ seic-enterprise      │ stopped      │ ~/Code/seic-enterprise      │                            ≈
├──────────────────────┼──────────────┼─────────────────────────────┼─────────────────────────── ≈
│ tryddevproject-21037 │ running (ok) │ ~/Code/tryddevproject-21037 │ https://tryddevproject-210 ≈
├──────────────────────┼──────────────┼─────────────────────────────┼─────────────────────────── ≈
│ Router               │ stopped      │ ~/.ddev                     │ http://localhost:9999      ≈
└──────────────────────┴──────────────┴─────────────────────────────┴─────────────────────────── ≈

┌───────────────────────────────────────────────────────────────────────────────────────────────┐
│ Project: tryddevproject-21037 ~/Code/tryddevproject-21037 https://tryddevproject-21037.ddev.s │
│ ite                                                                                           │
│ Docker platform: docker-desktop                                                               │
│ Router: traefik                                                                               │
├──────────┬──────┬──────────────────────────────────────────────────────┬──────────────────────┤
│ SERVICE  │ STAT │ URL/PORT                                             │ INFO                 │
├──────────┼──────┼──────────────────────────────────────────────────────┼──────────────────────┤
│ web      │ OK   │ https://tryddevproject-21037.ddev.site               │ php PHP8.1           │
│          │      │ InDocker: web:443,80,8025                            │ nginx-fpm            │
│          │      │ Host: 127.0.0.1:53226,53225                          │ docroot:'web'        │
│          │      │                                                      │ Mutagen enabled (ok) │
│          │      │                                                      │ NodeJS:18            │
├──────────┼──────┼──────────────────────────────────────────────────────┼──────────────────────┤
│ db       │ OK   │ InDocker: db:3306                                    │ mariadb:10.4         │
│          │      │ Host: 127.0.0.1:53227                                │ User/Pass: 'db/db'   │
│          │      │                                                      │ or 'root/root'       │
├──────────┼──────┼──────────────────────────────────────────────────────┼──────────────────────┤
│ Mailhog  │      │ MailHog: https://tryddevproject-21037.ddev.site:8026 │                      │
│          │      │ `ddev launch -m`                                     │                      │
├──────────┼──────┼──────────────────────────────────────────────────────┼──────────────────────┤
│ All URLs │      │ https://tryddevproject-21037.ddev.site,              │                      │
│          │      │ https://127.0.0.1:53225,                             │                      │
│          │      │ http://tryddevproject-21037.ddev.site,               │                      │
│          │      │ http://127.0.0.1:53226                               │                      │
└──────────┴──────┴──────────────────────────────────────────────────────┴──────────────────────┘
 
============= ddev-tryddevproject-21037-web healtcheck run =========
+ for f in /etc/bashrc/*.bashrc
+ source /etc/bashrc/commandline-addons.bashrc
++ export 'PATH=~/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/var/www/html/vendor/bin:/var/www/html/bin:/var/www/html/vendor/bin'
++ PATH='~/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/var/www/html/vendor/bin:/var/www/html/bin:/var/www/html/vendor/bin'
++ '[' -s /home/kdoyle/.nvm/nvm.sh ']'
++ '[' -s /home/kdoyle/.nvm/bash_completion ']'
++ '[' php = python ']'
++ '[' php = django4 ']'
++ ls '/home/kdoyle/.bashrc.d/*'
bash: rm -f /tmp/healthy && /healthcheck.sh: No such file or directory
============= ddev logs =========
+ logpipe=/var/tmp/logpipe
+ [[ ! -p /var/tmp/logpipe ]]
+ exec /usr/bin/supervisord -n -c /etc/supervisor/supervisord-nginx-fpm.conf
2023-07-28 14:12:37,746 INFO Included extra file "/etc/supervisor/php-fpm.conf" during parsing
2023-07-28 14:12:37,746 INFO Included extra file "/etc/supervisor/conf.d/mailhog.conf" during parsing
2023-07-28 14:12:37,746 INFO Included extra file "/etc/supervisor/conf.d/supervisor.conf" during parsing
2023-07-28 14:12:37,748 INFO RPC interface 'supervisor' initialized
2023-07-28 14:12:37,748 CRIT Server 'unix_http_server' running without any HTTP authentication checking
2023-07-28 14:12:37,748 INFO supervisord started with pid 126
2023-07-28 14:12:38,755 INFO spawned: 'child_exit_monitor' with pid 2038
2023-07-28 14:12:38,756 INFO spawned: 'php-fpm' with pid 2039
2023-07-28 14:12:38,757 INFO spawned: 'nginx' with pid 2040
2023-07-28 14:12:38,758 INFO spawned: 'mailhog' with pid 2041
[28-Jul-2023 14:12:38] NOTICE: fpm is running, pid 2039
[28-Jul-2023 14:12:38] NOTICE: ready to handle connections
[28-Jul-2023 14:12:38] NOTICE: systemd monitor interval set to 10000ms
2023-07-28 14:12:39,809 INFO success: child_exit_monitor entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-07-28 14:12:39,809 INFO success: php-fpm entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-07-28 14:12:39,809 INFO success: nginx entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
2023-07-28 14:12:39,809 INFO success: mailhog entered RUNNING state, process has stayed up for > than 1 seconds (startsecs)
============= contents of /mnt/ddev_config  =========
Error response from daemon: No such container: ddev-d9-db
======== Curl of site from inside container:
HTTP/1.1 200 OK
Server: nginx
Date: Fri, 28 Jul 2023 14:12:43 GMT
Content-Type: text/html; charset=UTF-8
Connection: keep-alive
Vary: Accept-Encoding

======== curl -I of http://tryddevproject-21037.ddev.site from outside:
curl: (7) Failed to connect to tryddevproject-21037.ddev.site port 80 after 4 ms: Couldn't connect to server
Unable to curl the requested project Please provide this output in a new gist at gist.github.com.
```
</details>         

### Expected Behavior

Expected containers to start up

### Actual Behavior

Error installing packages in the container

**Comment by rfay** [comment]
Hi @keithdoyle9 - 

What's going on is your Docker is unable to properly create/update the Traefik router container. It looks to me like a networking issue, I'm not sure. As you saw, I see

```
#6 0.296 fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/main/aarch64/APKINDEX.tar.gz
#6 0.733 fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/community/aarch64/APKINDEX.tar.gz
#6 0.735 60900EBEFFFF0000:error:0A000086:SSL routines:tls_post_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1889:
#6 0.735 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/main: Permission denied
#6 1.100 60900EBEFFFF0000:error:0A000086:SSL routines:tls_post_process_server_certificate:certificate verify failed:ssl/statem/statem_clnt.c:1889:
#6 1.100 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/community: Permission denied
#6 1.100 ERROR: unable to select packages:
#6 1.100   bash (no such package):
#6 1.100     required by: world[bash]
#6 1.100   curl (no such package):
#6 1.100     required by: world[curl]
#6 1.100   htop (no such package):
#6 1.100     required by: world[htop]
#6 1.100   vim (no such package):
#6 1.100     required by: world[vim]
#6 ERROR: process "/bin/sh -c apk add bash curl htop vim" did not complete successfully: exit code: 4
```

I would first `ddev poweroff` then `docker rm -f $(docker ps -aq)`, then restart Docker, then try again, please let me know what happens. 

You can absolutely return to the legacy router with `ddev config global --router=nginx-proxy` and get past this that way. 

Let me know what you figure out. And feel free to come over to Discord and we can do a live interactive debugging session. I sure want to know what's going on here. https://discord.gg/hCZFfAMc5k

(Traefik has been an option in DDEV for 7-8 months without anybody reporting this.)

**Comment by rfay** [comment]
When I've seen problems like this before they've been docker problems, often resolved by deleting all Docker images (which does no damage, they just have to be downloaded again).

```
docker rm -f $(docker ps -aq)
docker rmi -f $(docker images -q)
```

**Comment by keithdoyle9** [comment]
@rfay 

I think it's yet another self-signed cert issue. Our company injects it's own certs and we've run into this problem before but not at this step. I thought I had this remedied by importing the *.crt files in the container build process. 

It worked before upgrading to v1.22.0. Should I handle this differently for the upgrade?

From Dockerfile
```
ADD *.crt /usr/local/share/ca-certificates/
RUN update-ca-certificates --fresh
```

<img width="288" alt="image" src="https://github.com/ddev/ddev/assets/54675374/a8c7a0f7-6f00-4e24-b7e5-4e27dbda92c5">

**Comment by rfay** [comment]
I think this is for you and your company's IT department to sort out. You're on a VPN I guess. This problem is happening at docker build time.  it's not anything that DDEV has control of. You wouldn't be adding crt files into the docker container, it's docker itself that needs to figure this out, for the build stage. Poking around on Google gives some possibilities, but essentially you're looking at Docker configuration.

**Comment by keithdoyle9** [comment]
Thanks @rfay. When I work with our IT team I'll post whatever solution is figured out here.

**Comment by rfay** [comment]
I note that this can also be caused by network trouble and it looks the same; may also be upstream server trouble:

```
#6 0.151 fetch https://dl-cdn.alpinelinux.org/alpine/v3.18/main/aarch64/APKINDEX.tar.gz
  | #6 0.439 WARNING: updating and opening https://dl-cdn.alpinelinux.org/alpine/v3.18/main: IO ERROR
 ```

Here the attempt to access the alpinelinux CDN failed

**Comment by rfay** [comment]
I don't understand why you would have this problem only on the *traefik router* image which is Alpine Linux. I wonder if your company's setup just doesn't like the CDN they're using, `dl-cdn.alpinelinux.org`. 

What do you get on the host with 
```
curl -I https://dl-cdn.alpinelinux.org/alpine/v3.18/main/aarch64/APKINDEX.tar.gz
```

And what do you get inside any docker container? For example, 
```
ddev exec curl -I https://dl-cdn.alpinelinux.org/alpine/v3.18/main/aarch64/APKINDEX.tar.gz
```

One thing we could do is to build our own derivative image instead of using the Traefik image directly, and then you'd be getting everything from hub.docker.com, which apparently works for you already.

**Comment by rfay** [comment]
Since this seems to be a problem with your VPN, I can't guarantee that it's fixed now, but in v1.22.2 in a couple of weeks you won't be doing a build against the apk repositories, which seemed to be your problem. Instead you'll just be pulling from dockerhub.
 
I'd love it if you'd test, see https://ddev.readthedocs.io/en/latest/developers/building-contributing/#testing-latest-commits-on-head
